HT6_3.py

```diff
@@ -0,0 +1,240 @@
+import os
+import random
+import datetime
+import normalization.HT6 as norm
+import typing
+
+
+class Publication:
+    def __init__(self, publication_type: str, publication_body: str = "") -> None:
+        self.publication_type = publication_type
+        self.publication_body = publication_body
+
+
+class News(Publication):
+    def __init__(self, publish_datetime, publication_type, publication_body: str = "", location: str = "") -> None:
+        super().__init__(publication_type=publication_type, publication_body=publication_body)

```

Comment:
TO REVIEW:
I think it is worth to not provide publication type while creating object of the class, let's just assign certain pub type

class News(Publication):
    def __init__(self, publish_datetime, publication_body: str = "", location: str = "") -> None:
        super().__init__(publication_type="News", publication_body=publication_body)
.....

Reply:
The same for Ad and Joke